### DISCUSSION TITLE
Create a Modal when game ends

### DISCUSSION THREAD

**Discussion by ashishpapanai** [ashishpapanai/chessJS#101] [Ideas]
Currently, we use browser alerts to tell the player if Game's Over by checkmate, or the game is drawn because of 3 fold repetition or 50 moves rule etc. 

**We want to switch to Modal based alerts in order to give a personalised alert** to the user matching the theme of the website, We have already created a template model in `js/model.js` this can be used as a reference and new models for Victory, Loss or Draw can be created in a similar fashion.

**Comment by imakash3011** [comment]
I would like to work on this issue.

Is the reference template is js/modal.js because js/model.js is not in present js folder.

**Comment by ashishpapanai** [comment]
js/modal.js the modal file you just have to trigger it in the places where we are displaying a browser alert.
